frontend/src/metabase/visualizations/components/LegendHeader.jsx

```diff
@@ -26,6 +26,11 @@ export default class LegendHeader extends Component {
     actionButtons: PropTypes.node,
     description: PropTypes.string,
     classNameWidgets: PropTypes.string,
+    icon: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      color: PropTypes.string,
+      size: PropTypes.number,
+    }),
```

Comment:
Seeing this repeated makes me think of declaring it somewhere and spreading it :)

Reply:
Good point, probably worth defining a `iconPropTypes` constant and just import it everywhere